cmd/prysmctl/testnet/generate_genesis.go

```diff
@@ -238,6 +257,7 @@ func generateGenesis(ctx context.Context) (*ethpb.BeaconState, error) {
 	genesisTime := generateGenesisStateFlags.GenesisTime
 	numValidators := generateGenesisStateFlags.NumValidators
 	depositJsonFile := generateGenesisStateFlags.DepositJsonFile
+	eth1Data := generateGenesisStateFlags.OverrideEth1Data
```

Comment:
should call this overrideEth1Data, it's verbose but I think it's more readable instead of thinking it's actually the eth1Data object

Reply:
Definitely clearer